docs/en/engines/table-engines/mergetree-family/mergetree.md

```diff
@@ -1294,6 +1294,54 @@ In this sample configuration:
 </clickhouse>
 ```
 
+## CFS storage {#cfs-storage}
+
+First mount the remote CFS path to the local host such as `/mnt/cfs`.
```

Comment:
I would add at least one sentence about what is CFS (even link to wikipedia) and mention a couple: NFS, CephFS, Lustre

Reply:
I added a few simple sentences